src/components/customers/CustomerUsage.tsx

```diff
@@ -39,7 +39,7 @@ interface CustomerUsageProps {
 export const CustomerUsage = ({ id }: CustomerUsageProps) => {
   const { translate } = useInternationalization()
   const { data, error, loading, refetch } = useCustomerUsageQuery({
-    variables: { customerId: id },
+    variables: { customerId: id, subscriptionId: 'TODO' },
```

Comment:
Isn't the subscriptionId the id passed as props in this component ?

Reply:
I'm working on it right now, it was just so the codegen will not break